virttest/utils_misc.py

```diff
@@ -2093,7 +2093,7 @@ def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
     """
     Get the machine type the host support,return a list of machine type
     """
-    o = process.system_output("%s -M ?" % qemu_binary)
+    o = process.system_output("%s -M \?" % qemu_binary)
```

Comment:
I'm not quite sure. But since this line did not specify `shell`, could this slash cause over-escaping?